docs/source/user/programming-framework/a-framework-to-understand-dp.ipynb

```diff
@@ -471,32 +497,34 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "We let $MI$ be `AbsoluteDistance` and $MO$ be `MaxDivergence`.\n",
-    "It can be shown that for any choice of $u$ and $v$ such that $d_{Sym}(u, v) \\leq d_{in}$, and $d_{out} = d_{in} / scale$, then:\n",
-    "$$d_{MaxDivergence}(u, v) \\leq d_{out}$$\n",
+    "It can be shown that for any choice of $u, v \\in \\mathbb{R}$ such that $d_{\\mathrm{Abs}}(u, v) \\leq d_{in}$, and $d_{out} = d_{in} / scale$, then:\n",
+    "$$D_{\\mathrm{MaxDivergence}}(\\mathrm{laplace\\_noise}(u), \\mathrm{laplace\\_noise}(v)) \\leq d_{out}$$\n",
     "Therefore, if the data types in this function had infinite precision, then $laplace\\_noise$ would be a measurement.\n",
     "Other common metrics to express sensitivities are `L1Distance` and `L2Distance`."
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Definition of Stability\n",
     "Similar to how we defined the privacy of a measurement $M(\\cdot)$, we can also define the stability of a transformation, $T(\\cdot)$:\n",
     "\n",
-    "> $T(\\cdot)$ is $d_{out}$-stable with respect to input metric $MI$ and output metric $MO$ if, for any choice of datasets $u$ and $v$ such that $d_{MI}(u, v) \\leq d_{in}$, we have that $d_{MO}(T(u), T(v)) \\leq d_{out}$.\n",
+    "> $T(\\cdot)$ is $\\bm{(d_{in}, d_{out})}\\textbf{-stable}$ with respect to input metric $MI$ and output metric $MO$ if, for any choice of datasets $u$ and $v$ such that $d_{MI}(u, v) \\leq d_{in}$, we have that $d_{MO}(T(u), T(v)) \\leq d_{out}$.\n",
```

Comment:
I believe 

$$d_{MO}(T(u), T(v)) \\leq d_{out}$$ 

should now read 

$$D_{MO}(T(u), T(v)) \\leq d_{out}$$ 

following a similar change on the _Definition of Privacy_ section

```suggestion
    "> $T(\\cdot)$ is $\\bm{(d_{in}, d_{out})}\\textbf{-stable}$ with respect to input metric $MI$ and output metric $MO$ if, for any choice of datasets $u$ and $v$ such that $d_{MI}(u, v) \\leq d_{in}$, we have that $D_{MO}(T(u), T(v)) \\leq d_{out}$.\n",
```

Reply:
This is pretty subtle-- but the notation for metrics is lower-case, and the notation for measures is upper-case. Since we're dealing with metrics at this point, it should remain lower-case.